pkg/proxy/proxier.go

```diff
@@ -715,9 +793,14 @@ func iptablesCommonPortalArgs(destIP net.IP, destPort int, protocol api.Protocol
 		"--comment", service.String(),
 		"-p", strings.ToLower(string(protocol)),
 		"-m", strings.ToLower(string(protocol)),
-		"-d", fmt.Sprintf("%s/32", destIP.String()),
-		"--dport", fmt.Sprintf("%d", destPort),
 	}
+
+	if destIP != nil {
+		args = append(args, "-d", fmt.Sprintf("%s/32", destIP.String()))
+	}
+
+	args = append(args, "--dport", fmt.Sprintf("%d", destPort))
```

Comment:
The format needs to match, but the order does not - that's (unfortunately) hopeless, so ancient iptables args-lists are sorted and compared.  Puke.